cmd/main.go

```diff
@@ -103,6 +110,8 @@ func main() {
 		"The controller will load its initial configuration from this file. "+
 			"Command-line flags will override any configurations set in this file. "+
 			"Omit this flag to use the default configuration values.")
+	flag.BoolVar(&enableGangScheduling, "enable-gang-scheduling", false, "Enable gang-scheduling for a group of lws pods [ 1 leader pod + (size-1) worker pods].")
```

Comment:
I think we encourage users to use the config file instead, so maybe we don't need this here any more.